src/run/blake2b.rs

```diff
@@ -47,30 +47,24 @@ impl Clone for Blake2b {
     }
 }
 
-pub fn blake2bf_instruction(buf: Buffer) -> Buffer {
+pub fn blake2bf_instruction(buf: Buffer) -> Option<Buffer> {
     let mut b = [0u8; 213];
     for i in 0..b.len() {
         b[i] = buf.read_byte(i as u128);
     }
     b[212] = b[212] & 0x1u8;
```

Comment:
the current implementation for node (and onestepproof) also treat as error:
* buffer with more than 213 bytes
* if byte 212 ("f" flag) is > 1